deploy/manifests/base/deployment.yaml

```diff
@@ -38,6 +38,11 @@ spec:
             - name: identity
               mountPath: /data/.autoretrieve/peerkey
               subPath: peerkey
+          resources:
+            limits:
+              memory: 32Gi
+            requests:
+              memory: 32Gi
```

Comment:
* This should move to the dev overlay as a patch. 
* The physical memory available on AWS EC2 instance types jumps from 32 to 64 GiB. Whichever type we choose, that node will need to run other administrative pods like the logs collector etc. So not all of its memory will be available for use by other pods. If we absolutely need 32 GiB of memory for auto retrieve then we need to pick an instance type that has 64 GiB of memor; the cheapest with that much memory is `x2gd.xlarge` at `$0.334` per hour. If we can reduce the memory requirements of autoretrieve to say 25-ish GiB then we can use an instance type with 32 GiB of memory, the cheapest of which is `r5a.xlarge` at `$0.226` per hour. See pricing [here](https://aws.amazon.com/ec2/pricing/on-demand/). @willscott  what are your thoughts?
* While at it, I think it's also a good idea to set CPU limit

Reply:
x2gd.xlarge is fine for now.

Reply:
Went with `r5a-2xl`, the next cheapest with 64 GiB that runs on x86_64 CPU arch since the `gd` instance types uses the ARM-based Graviton 2.

worker node is up. The remaining comments I made above are not blockers. Please feel free to work on them in follow up PRs 👍